server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/wallets/cryptx/v2/CryptXWallet.java

```diff
@@ -110,7 +110,7 @@ public String getCryptoAddress(String cryptoCurrency) {
         coins.add(CryptoCurrency.BCH.getCode());
         coins.add(CryptoCurrency.ETH.getCode());
         coins.add(CryptoCurrency.USDT.getCode());
-
+        coins.add(CryptoCurrency.BTBS.getCode());
```

Comment:
Does CryptX support BTBS? Their documentation does not mention it: 
https://api.cryptx.com/#section/Coin-Digital-Currency-Support